composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~4.0",
-        "orchestra/testbench": "~3.0"
+        "orchestra/testbench-browser-kit": "~3.4"
```

Comment:
I have restored the original testbench package. Turned out that the interaction with session functionality has been moved to testbench-browser-kit, so I decided to use the `InteractsWithSession` trait directly where required.